File: CSharp/LINQtoDB/Models/Department.cs
using LinqToDB.Mapping;

namespace Models;

[Table("departments")]
public class Department
{
    [PrimaryKey, Identity, Column("id")]
    public int Id { get; set; }

    [Column("name"), NotNull]
    public string? Name { get; set; }

    public override string ToString() => $"{{ Department: {{ Id: {Id}, Name: {Name} }} }}";
}

File: FSharp/LINQtoDB/Models/Department.fs
namespace Models

open LinqToDB.Mapping

[<Table "departments">]
type Department (name) =
  new () = Department ("")

  [<PrimaryKey>][<Identity>][<Column "id">]
  member val Id = 0 with get, set

  [<Column "name">][<NotNull>]
  member val Name = name with get, set

  with override this.ToString () =
        $"{{ Department: {{ Id: {this.Id}, Name: {this.Name} }} }}"
